app/networkapi/templates/shared/adminsortable_objects_custom.html

```diff
@@ -4,7 +4,7 @@
     <input name="pk" type="hidden" value="{{ object.pk }}" />
     <div id="sortable-object">
         <a href="{% url opts|admin_urlname:'do_sorting' object.model_type_id %}" class="admin_sorting_url">
-            <img class="person-image" src="{{ object.image.url }}">
+            <img class="instance-image" src="{{ object.image.url }}">
```

Comment:
I actually think we shouldn't genericize this file because it relies on properties that may or may not be part of a model (for e.g. `image` and also soon to be added - `featured`). So, I think it makes sense to duplicate this file for each app.